packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -60,30 +66,47 @@ export const MenuItem = ({ children, onClick, to, isFocused, ...props }) => {
     }
   };
 
-  return to ? (
-    <OptionLink to={to} {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionLink>
-  ) : (
-    <OptionButton onKeyDown={handleKeyDown} onMouseDown={onClick} type="button" {...menuItemProps}>
-      <Box padding={2}>
-        <Typography>{children}</Typography>
-      </Box>
-    </OptionButton>
-  );
+  if (to)
+    return (
+      <OptionLink to={to} {...menuItemProps}>
+        <Box padding={2}>
+          <Typography>{children}</Typography>
+        </Box>
+      </OptionLink>
+    );
+  else if (href && isExternal)
+    return (
+      <OptionExternalLink {...menuItemProps}>
+        <Box padding={2}>
+          <Link href={href} isExternal={isExternal}>
```

Comment:
following my previous comment, wdyt about: 
```js
    <OptionExternalLink href={href} isExternal={isExternal} {...menuItemProps}>
      <Box padding={2}>
        <Typography>{children}</Typography>
      </Box>
    </OptionExternalLink>
```

Reply:
Ya we can do that.
Please reply to above comment so we can proceed with final solution.